config/test.exs

```diff
@@ -93,3 +93,14 @@ config :trento, Trento.Infrastructure.SoftwareUpdates.MockSuma,
   ]
 
 config :trento, :activity_log, refresh_interval: 1
+
+config :junit_formatter,
+  report_file: "report_file_test_#{DateTime.now!("Etc/UTC")}.xml",
+  report_dir: "/tmp",
+  print_report_file: true,
+  prepend_project_name?: true,
+  include_filename?: true
+
+config :trento,
+       :flaky_tests_detection,
+       enabled?: System.get_env("WRITE_JUNIT") == "1" || false
```

Comment:
```suggestion
       enabled?: System.get_env("WRITE_JUNIT") == "1"
```

Setting a default flag for a boolean option is redundant as it will be `false` in any case :smile: